quality-assurance/test-strategy.md

```diff
@@ -84,9 +95,11 @@ The disaster recovery plan should be tested for a new or significantly changed s
 
 ### Functional
 
-![Test Pyramid](images/test-pyramid.png)
+![Test Pyramid](images/the_test_pyramid.png)
```

Comment:
Nice picture, but we should be careful not to suggest that this is a definitive diagram.  This is only an example for a particular project.  Can you add some generic statement about test pyramids?

Reply:
I dislike the dev/tester/user part of the pyramid